codebases/signalen.html

```diff
@@ -0,0 +1,136 @@
+---
+layout: codebase
+title: Codebases we work with - Signalen
+---
+
+<div class="cb-section">
+  <div>
+    <h1 class="title">Codebases we work with</h1>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-maturity-detail-header">
+    <div class="cb-maturity-detail-title cb-maturity-s1">
+      <div class="cb-logo"><img src="../photos/codebases/signalen/signalen_logo_small.png" alt="Signalen logo"></div>
+      <h1>Signalen</h1>
+    </div>
+    <div class="cb-maturity-stage cb-maturity-s1">Stage 1: <strong>Assessment</strong></div>
+  </div>
+
+  <div class="cb-detail-intro">
+    <div class="cb-detail-info">
+      <h3>What it is</h3>
+      <p>Signalen makes it easier for residents to make a report, and helps the city respond to the most important things first.</p>
+      <p><strong>It includes:</strong></p>
+      <ul>
+        <li>Algorithmic processing by type of complaint, and second layer triaging and prioritisation by humans</li>
+        <li>Automatic updates to complainants about the status of their report</li>
+        <li>Automatic alerts to city staff based on analysis of combined reports</li>
+        <li>Allowing reports to be passed on to other city workers and subcontractors</li>
+      </ul>
+    </div>
+
+    <div class="cb-detail-gallery">
+      <a href="#img1">
+        <img class="lb-thumb" src="../photos/codebases/signalen/signalen_thumb_g01.png" alt="Signalen product thumbnail">
+      </a>
+      <!-- lightbox container hidden with CSS -->
+      <div class="lightbox" id="img1">
+        <a href="#img3" class="lb-btn lb-btn-prev"></a>
+        <a href="#" class="lb-btn lb-btn-close"></a>
+        <a href="#img2"><img src="../photos/codebases/signalen/signalen_g01.png" alt="Signalen product example 1"></a>
+        <a href="#img2" class="lb-btn lb-btn-next"></a>
+      </div>
+      <!-- lightbox container hidden with CSS -->
+      <div class="lightbox" id="img2">
+        <a href="#img1" class="lb-btn lb-btn-prev"></a>
+        <a href="#" class="lb-btn lb-btn-close"></a>
+        <a href="#img3"><img src="../photos/codebases/signalen/signalen_g02.png" alt="Signalen product example 2"></a>
+        <a href="#img3" class="lb-btn lb-btn-next"></a>
+      </div>
+      <!-- lightbox container hidden with CSS -->
+      <div class="lightbox" id="img3">
+        <a href="#img2" class="lb-btn lb-btn-prev"></a>
+        <a href="#" class="lb-btn lb-btn-close"></a>
+        <a href="#img1"><img src="../photos/codebases/signalen/signalen_g03.png" alt="Signalen product example 3"></a>
+        <a href="#img1" class="lb-btn lb-btn-next"></a>
+      </div>
+    </div>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>How we're helping</h3>
+    <p>We're helping Amsterdam and the VNG define a governance structure, including shared feature roadmaps and technical steering.</p>
```

Comment:
This is the first time we use VNG on this page, should we explain the abbreviation?

Reply:
Thanks, I've fixed this!